x-pack/test/plugin_api_perf/test_suites/task_manager/task_manager_perf_integration.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import supertestAsPromised from 'supertest-as-promised';
+
+export default function({ getService }: { getService: (service: string) => any }) {
+  const log = getService('log');
+  const config = getService('config');
+  const supertest = supertestAsPromised(url.format(config.get('servers.kibana')));
```

Comment:
What is the difference between this and `getService('supertest');`?

Reply:
I don't actually know, this is copied from the exiting integration test.
They were originally in the same file and I extracted this to it's own folder, so copied the scaffolding.

I'll double check why we do it this way in those tests.

Reply:
Turned out this was needed in the old js code, but not the new ts versions, so this is now a `getService('supertest')`